scripts/ci/libraries/_start_end.sh

```diff
@@ -16,6 +16,26 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Starts group for Github Actions - makes logs much more readable
+function start_end::group_start {
+    if [[ ${GITHUB_ACTIONS=} == "true" ]]; then
```

Comment:
```suggestion
    if [[ ${GITHUB_ACTIONS} == "true" ]]; then
```

Should it be like this instead? The same for the one below in `function start_end::group_end` as well?

Reply:
Nope. It's OK. We have 'set -u` (fail on undefined) set in the script so it will fail if a variable is undefined. It helps to catch typos but in this case GITHUB_ACTIONS might be undefined when run locally so I (out of precaution) add = to make the variable value empty rather than undefined.